core/src/com/unciv/UncivGame.kt

```diff
@@ -159,6 +171,17 @@ class UncivGame(parameters: UncivGameParameters) : Game() {
         super.resume()
         musicController.resume()
         if (!isInitialized) return // The stuff from Create() is still happening, so the main screen will load eventually
+
+        // This is also needed in resume to open links and notifications
+        // correctly when the app was already running. The handling in onCreate
+        // does not seem to be enough
+        if (deepLinkedMultiplayerGame != null) {
```

Comment:
Since this is called twice how about making it a function, tryLoadDeepLinkedGame() or something?